how-to/how-to-use-sysctls-with-kata.md

```diff
@@ -0,0 +1,128 @@
+# Setting Sysctls with Kata
+
+## Sysctls
+In Linux, the sysctl interface allows an administrator to modify kernel 
+parameters at runtime. Parameters are available via the /proc/sys/ virtual 
+process file system. 
+
+The parameters include the following subsystems among others:
+- kernel (kernel.)
+- networking (net.)
+- virtual memory (vm.)
+- file systems (fs.)
+
+To get a complete list of kernel parameters, you can run:
+```
+sudo sysctl -a
+```
+
+Both Docker and Kubernetes provide mechanism for setting namespaced sysctls. 
+Namespaced sysctls can be set per pod in case of Kubernetes or per container
+in case of Docker.
+The following sysctls are known to be namespaced and can be set with 
+Docker and Kubernetes:
+
+- kernel.shm*,
+- kernel.msg*,
+- kernel.sem,
+- fs.mqueue.*,
+- net.*.
+
+### Namespaced Sysctls:
+
+Kata Containers supports setting namespaced sysctls with Docker and Kubernetes.
+All namespaced sysctls can be set in the same way as regular Linux based
+containers, the difference being, in case of Kata they are set inside the guest.
+
+#### Setting Namespaced Sysctls with Docker:
+
+```
+$ sudo docker run --runtime=kata-runtime  --sysctl fs.mqueue.queues_max=512  --sysctl kernel.shmmax=1024  -it alpine sh
+# cat /proc/sys/fs/mqueue/queues_max 
+512
+# cat /proc/sys/kernel/shmmax
+1024
+```
+For additional documentation on setting sysctls with Docker please refer to [Docker-sysctl-doc](https://docs.docker.com/engine/reference/commandline/run/#configure-namespaced-kernel-parameters-sysctls-at-runtime).
+
+
+#### Setting Namespaced Sysctls with Kubernetes:
+
+Kubernetes considers certain sysctls as safe and others as unsafe. For detailed
+information about what sysctls are considered unsafe, please refer to the [Kubernetes sysctl docs](https://kubernetes.io/docs/tasks/administer-cluster/sysctl-cluster/).
+For using unsafe systcls, the cluster admin would need to allow these as:
+
+```
+kubelet --allowed-unsafe-sysctls \
```

Comment:
- Missing `$ `.
- Does this need a `sudo` prefix or does `kubelet` handle that?
- Don't use backslashes - again see https://github.com/kata-containers/documentation/blob/master/Documentation-Requirements.md#code-blocks.

Reply:
Fixed. No this does not need sudo, kubelet handles it.